site/content/posts/2022-04-15-fips-and-more.md

```diff
@@ -0,0 +1,64 @@
+
+**Note: We will not provide official support for FIPS configuration, and will not respond to GitHub issues opened related to FIPS support. Out intent is to provide you with an example of how you can do this yourself in your environments.**  
+
+## Supervisor with default HTTPS listener port
+
+With [v0.13.0](https://github.com/vmware-tanzu/pinniped/releases/tag/v0.13.0) we had announced that we will disable the use of default HTTP listeners.
+
+**Breaking change in this release: With this release, we disable the Supervisor's HTTP listener by default, and do not allow it to be configured to bind to anything other than loopback interfaces.**
+
+However, we do recognize that it may take some users time to adjust to this breaking change. If you want to bring back the insecure http listen ports behavior into your deployments, you can set the  variable, ***deprecated_insecure_accept_external_unencrypted_http_requests***, in your Supervisor deployment yaml file. This will print a warning in the pod logs that lets you know that this is an insecure option. Please do note that we plan to deprecate this field in some future release and only provide you with secure https options. We communicate any API changes atleast 2 releases prior to the change, so you will have plenty of time to make changes to your system.  
+
+This feature does not change any HTTPS listen port configuration nor does it change the user's ability to do the following with the HTTP listener ports:
+
+1. Enable or disable the HTTP listening port
+2. Configure the HTTP listening port to listen on tcp loopback interfaces (ipv4, ipv6, or both) or on a unix domain socket file for listening for connections from inside the pod, for example connections from a service mesh's sidecar container
+3. Choose the port number for the HTTP listening port
+
+For more information on this feature refer to [#981](https://github.com/vmware-tanzu/pinniped/issues/981).
+
+## Workspace ONE Identity Provider configuration
+
+We continue to gather feedback from the community around the need to integrate with different Identity Providers. With this in mind, we have documented our support for configuring [VMware Workspace ONE Access](https://www.vmware.com/products/workspace-one/access.html) (formerly VMware Identity Manager) as an Identity provider. Workspace ONE access also acts as a broker to other identity stores and providers—including Active Directory (AD), Active Directory Federation Services (ADFS), Azure AD, Okta and Ping Identity to enable authentication across on-premises, software-as-a-service (SaaS), web and native applications. Available as a cloud-hosted service, Workspace ONE Access is an integral part of the Workspace ONE platform.
+
+Refer to our detailed guide  on [how to configure supervisor with Workspace ONE Access](({{< ref "docs/howto/configure-supervisor-with-workspace_one_access.md" >}}))  
```

Comment:
Link is not working, maybe this will fix it:

```suggestion
Refer to our detailed guide on [how to configure supervisor with Workspace ONE Access]({< ref "docs/howto/configure-supervisor-with-workspace_one_access.md" >}).
```

Reply:
Resolved